Improve sys logs page by clarifying options dropped (#41395)
kubernetes/website: content/en/docs/concepts/cluster-administration/system-logs.md

klog is the Kubernetes logging library. [klog](https://github.com/kubernetes/klog)
generates log messages for the Kubernetes system components.

> Kubernetes is in the process of simplifying logging in its components.

Reviewer: Is this still true?

The following klog command line flags

> [are deprecated](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
> starting with Kubernetes 1.23 and removed in 1.26:

Reviewer: ```suggestion
[were deprecated](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
in Kubernetes 1.23 and removed in Kubernetes 1.26:
```
